static/app/components/replays/useReplaysCount.tsx

```diff
@@ -43,17 +47,27 @@ function useReplaysCount({groupIds, transactionNames, organization, project}: Op
       );
     }
     if (groupIds && groupIds.length) {
+      // The endpoint only supports 25, that's also the
+      // max rows that the issue list page will show.
+      const chunkSize = 25;
+
+      const chunks = Math.ceil(groupIds.length / chunkSize);
+      const conditions: string[] = [];
+      for (let i = 0; i < chunks; i++) {
```

Comment:
can we cap this? worried about unbounded parallel queries that could happen

Reply:
I'm taking this out.

I've added a check so that we're only fetching counts for issues that we havn't seen yet. See `groupsToFetch` variable.

This check means that we'll fetch up to 25 issues at pageload, and then up to 25 more issues as polling happens and the issues list refreshes.